_posts/2021-05-08-PyFlames_2021.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "PyFlames 2021"
+authors: 
+  - rahul
+description: "Mega meetup"
+categories: [ meetup, talks ]
+image:
+featured: true
+hidden: true
```

Comment:
What's this `hidden`? 
Don't suppose we've any hidden flag.

Reply:
Resovled here: 
07d2a65